apps/server/src/modules/task/uc/submission.uc.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId, Permission, PermissionContextBuilder, Submission, User } from '@shared/domain';
+import { AuthorizationService } from '@src/modules/authorization';
+import { SubmissionService } from '../service/submission.service';
+
+@Injectable()
+export class SubmissionUC {
```

Comment:
We don't have consisting naming of UCs but most of them are written with a small c: NewsUc, SchoolUc, ... I for one like it better with the small c. I don't know if we have a rule for this.

Generally casing of acronyms is an often discussed topic, e.g.: https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Reply:
Die Diskussion wurde nie zu Ende geführt. Ich persönlich habe keine starke Meinung dazu. Und es schein tatsächlich je nach Entwickler fahl zu sein. Ich schlage vor so zu lassen und ich nehme das Thema mit ins Arc Meeting. Eventuell mit folge Ticket.

Reply:
renamed to submissionUc